src/components/useraccount/MyOrders.vue

```diff
@@ -63,11 +63,13 @@
                  class="col-sm-2 col-xs-4">
               <BaseMoney :money="order.totalPrice"/>
             </div>
-            <div class="col-sm-2 hidden-xs">
-              <span>{{ order.paymentState || "-" }}</span>
+            <div class="col-sm-2 hidden-xs"
+                 data-test="payment-state">
+              <span>{{ $t('paymentState.' + order.paymentState )}}</span>
             </div>
-            <div class="col-sm-2 hidden-xs">
-              <span>{{ order.shipmentState || "-" }}</span>
+            <div class="col-sm-2 hidden-xs"
+                 data-test="shipment-state">
+              <span>{{ $t('shipmentState.' + order.shipmentState )}}</span>
```

Comment:
actually, why wouldn't be there a translation available? there aren't other values than these

Reply:
For example, when the API decides to introduce a new value. Or when the API decides to remove a value. An app shouldn't rely on an API providing very specific values when it can be avoided, otherwise the API would never be able to grow without breaking many apps on its way.